packages/alchemy/package.json

```diff
@@ -56,8 +69,22 @@
     "zustand": "^4.5.2"
   },
   "peerDependencies": {
+    "@tanstack/react-query": "^5.28.9",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "viem": "2.8.6"
   },
+  "peerDependenciesMeta": {
+    "react": {
+      "optional": true
+    },
+    "react-dom": {
+      "optional": true
+    },
+    "@tanstack/react-query": {
+      "optional": true
+    }
+  },
```

Comment:
this is the way you're intended to set optional deps that are also peer dependencies

Reply:
oh interesting, should we move the optDeps into this too then at some point?

Reply:
In this case I need to pin the `react-query` dep to a specific version. But yea I think we should do that for the `aa-signers` repo. those optional deps probably should be optional peer deps instead of optional deps